erpnext/public/js/hub/pages/FeaturedItems.vue

```diff
@@ -33,10 +33,7 @@ export default {
 
 			// Constants
 			page_title: __('Your Featured Items'),
-            empty_state_message: __(`No featured items yet. Got to your 
-                                <a href="#marketplace/published-items">
-                                Published Items</a> 
-                                and feature upto 8 items that you want to highlight to your customers.`)
+            empty_state_message: __('No featured items yet. Got to your <a href="#marketplace/published-items">Published Items</a> and feature up to eight items that you want to highlight to your customers.')
```

Comment:
```suggestion
            empty_state_message: __('No featured items yet. Got to your {0} and feature up to eight items that you want to highlight to your customers.',
            	[`<a href="#marketplace/published-items">${__("Published Items")}</a>`])
```
Can move out HTML elements as variables